graphql_compiler/query_pagination/__init__.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import namedtuple
+
+from graphql.language.printer import print_ast
+
+from graphql_compiler.cost_estimation.cardinality_estimator import estimate_number_of_pages
+from graphql_compiler.query_pagination.parameter_generator import (
+    hydrate_parameters_of_parameterized_query
+)
+from graphql_compiler.query_pagination.query_parameterizer import generate_parameterized_queries
+
+
+PaginationQueries = namedtuple(
```

Comment:
Add a description of what this is. It's only obvious from the name if you've been working on it for a while.

Reply:
Some problems:
- The name is not the best, and I don't have a better idea.
- This class exists solely to better document the return value of a function, but in the `Returns` section of that function you end up explaining all over again what this means.
- You can't have `continuation_query` be `None`, but `continuation_parameters` to actually exist.

To me it seems more natural to group a query and parameters in one namedtuple (maybe call it `QueryWithParams`), and return a pair of `QueryWithParams` in your function, where the second one is optional.

Reply:
If we don't use `PaginationQueries`, and instead just use a tuple with two `QueryWithParams`, the user will have to index by integers: `result[0]` for the next page query, and `result[1]` for the continuation query. In my eyes, `result.next_page_query` and `result.continuation_query` is bound to raise less eyebrows months on after this code is written.

Reply:
You can unpack as `first_query, next_query = function_call(...`. I don't disagree. You can make it work if you document it well. I'm just giving a suggestion.

Reply:
Your unpacking idea is the best solution thus far, I'll use that one! Thanks.